src/Stripe.net/Entities/PaymentIntents/PaymentIntentTransferData.cs

```diff
@@ -6,7 +6,7 @@ namespace Stripe
     public class PaymentIntentTransferData : StripeEntity
     {
         [JsonProperty("amount")]
-        public long Amount { get; set; }
+        public long? Amount { get; set; }
```

Comment:
Is this a breaking change?

Reply:
In the strictest sense, yes: code that uses this property will probably need to be updated to handle the nullability. That said, given that very few people are using this, and anyone who does is probably seeing deserialization errors, I think it's fine to consider this a bug fix and release as a patch version (though the rest of the changes in this PR qualify this as a minor, not a patch).